services/knowledge_base/kb.py

```
@@ -0,0 +1,26 @@
+async def get_kb_items(user_id: str, page: int = 0):
+    try:
+        logger.info(f"Fetching items for user: {user_id}, page: {page}")
```

Ruff (0.8.2)

Undefined name `logger`

(F821)